www/img-status-lang.php

```diff
@@ -12,13 +12,13 @@
 
 $available_langs = revcheck_available_languages($idx);
 
+$lang = $_GET['lang'];
 $langs = array_keys($LANGUAGES);
```

Comment:
Where is this used?

Reply:
In the `in_array()` test and (possibly) call to `generate_image()` that immediately follows?

Reply:
I can't see any usage of this variable in this file. Is the file included somewhere else? If not, then this variable is dead code.

Reply:
Searching GitHub also confirms that this assignment can be deleted https://github.com/search?q=repo%3Aphp%2Fweb-doc%20%24langs&type=code